openfisca_core/scripts/migrations/v16_2_to_v17/xml_to_yaml.py

```diff
@@ -80,7 +80,13 @@ def transform_values_history(children, value_format):
         elif child.tag == 'END':
             values[date] = {'value': None}
         elif child.tag == 'PLACEHOLDER':
-            values[date] = "expected"
+            values[date] = 'expected'
+
+        if 'reference' in child.attrib:
+            if values[date] == 'expected':
+                values[date] = {'expected': None, 'reference': child.attrib['reference']}
```

Comment:
why is `'expected': None`? It seems to me that it would be more appropriate to have a syntax that says that the `value[date]`is **in fact** expected.

Reply:
The admissible syntaxes for expected parameters were decided here : https://github.com/openfisca/openfisca-core/issues/523#issuecomment-313648250